walletlibrary/src/main/java/com/microsoft/walletlibrary/wrapper/OpenIdResolver.kt

```diff
@@ -20,22 +20,22 @@ import com.microsoft.walletlibrary.util.VerifiedIdRequestFetchException
 object OpenIdResolver {
 
     // Fetches the presentation request from VC SDK using the url and converts it to raw request.
-    internal suspend fun getRequest(uri: String): OpenIdRawRequest {
+    internal suspend fun getRequest(uri: String): OpenIdProcessedRequest {
```

Comment:
Are the prefer header passed here. And looks like presentation service is a bit outdated as well.